Controls/Customer1.c

```diff
@@ -0,0 +1,27 @@
+#include "os.h"
+
+/**
+ * Customer 1 wants to buy something!
+ */
+
+
+/**
+ * @brief Get the FOURTH item and cost on the menu (Tasks.h). Print the value of this item,
+ * along with the corresponding cost. Add the cost to the business revenue and print the current business revenue. 
+ * 
+ * In this method, use POINTER MATH to access values in both arrays.
```

Comment:
Why pointer math? I don't think this is good practice when you can just index an array because it can get out of hand. Consider:
```C
// declared somewhere else
extern int arr[10];
int getNumber(void);

// some fun with arrays
int foo = getNumber();
int bar = arr[foo + 3];              // pretty readable
int baz = *(arr + foo + 3);          // less readable
int var = *(3 + arr + foo);          // even less readable
// I'm having fun now, and this isn't a serious example
int xyz = * (int *) ( foo * sizeof * arr + 12 + (char *) arr);
```

Reply:
The reason for the pointer math vs array indexing was that I wanted them to use pointers after they had just learned about them, and showing them that an array can be accessed through pointers as well as indexing seemed like a simple way to do it. 

That being said, it does make sense that we wouldn't want this in production code and we shouldn't be teaching it if we don't necessarily want them to use it in Controls code. Is there another challenge someone can think of that uses pointers that might be used in production code?

Reply:
You could make them write a function that returns multiple values (eg. if it was successful or not and some data).